CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Community Code of Conduct
+
+**Version 1.2  
```

Comment:
There is version mismatch in this file.
Kindly read [TRG](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-01/#code-of-conduct) 
The Version 2.0 of the Eclipse Foundation Community Code of Conduct was released on Jan 01, 2023.